### PR TITLE
Remove lambdas, formatting improvements

### PR DESCRIPTION
**Pull request by cody-littley**
Improves formatting and documentation. Removes lambda creation.

### CODE REVIEW

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WireBuilder.java [hunk 1]
```diff
@@ -77,6 +89,6 @@ public Wire<T> build() {
         if (concurrent) {
             return new ConcurrentWire<>(executor, consumer);
         }
-        return new SequentialWire<>(executor, consumer);
+        return new SequentialWire<>(consumer);
```
Comment (OlegMazurov) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/WireBuilder.java:92]: There should be no conceptual difference between `SequentialWire` and `ConcurrentWire`. I thought either both should assume there are executed in a ForkJoinPool, which is my preference, or both accept `Executor`.

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/SequentialWire.java [hunk 2]
```diff
@@ -13,12 +29,16 @@
  */
 public class SequentialWire<T> implements Wire<T> {
     private final Consumer<T> consumer;
-    private final AtomicReference<Task> lastTask = new AtomicReference<>(new Task(true));
+    private final AtomicReference<Task<T>> lastTask;
```
Comment (alittley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/SequentialWire.java:32]: Consider renaming to `previousTask`, as "last" ambiguously can mean either "previous" or "final"

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/Task.java [hunk 3]
```diff
@@ -1,53 +1,106 @@
...
      */
-    public Task() {
-        this(false);
-    }
+    private final Consumer<T> consumer;
+
+    /**
+     * The data that this task will send to the consumer.
+     */
+    private T t;
```
Comment (alittley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/Task.java:58]: consider renaming to `data` or `taskData`

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/Task.java [hunk 4]
```diff
@@ -1,53 +1,106 @@
...
+     * {@inheritDoc}
+     */
     @Override
-    protected void setRawResult(Void value) {
-    }
+    protected void setRawResult(Void value) {}
 
     /**
-     * Send data
+     * Provide the object to be sent to the consumer and set the next task to that will follow this task.
```
Comment (alittley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/Task.java:86]: incorrect grammar "set the next task to that will follow"

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/Task.java [hunk 5]
```diff
@@ -1,53 +1,106 @@
...
+     *
+     * @param nextTask the task that comes after this task
```
Comment (alittley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/internal/Task.java:88]: param doc ordering is reversed from param order

**Review by alittley — APPROVED** [review]

**Pull request closed without merging**